source/index.html.erb

```diff
@@ -50,30 +49,30 @@ title: Index Page Example
       <section class="content-section content-section--with-top-border">
         <h2>Keep your existing infrastructure</h2>
         <p class="content-section__body">
-         You won't have to change your infrastructure or current Wifi provider.<br>
+         You won't have to change your infrastructure or current Wifi provider.
          GovWifi just makes all participating wifi networks appear as one to users.
         </p>
         </section>
    
 
         <section class="content-section content-section--with-top-border">
         <h2>Designed to meet government security standards</h2>
-          <ul>
           <p>
             GovWifi is tested to keep users safe with features like:</p>
+          <ul>
```

Comment:
This site doesn't actually use the [GOV.UK design system](https://design-system.service.gov.uk/), so we can't use their styling here - we'll need to either custom style this or find out how other people did it.

Do you know any other product pages that use bulleted lists?